cinema/tests/test_movie_api.py

```diff
@@ -43,9 +43,7 @@ def sample_actor(**params):
 
 
 def sample_movie_session(**params):
-    cinema_hall = CinemaHall.objects.create(
-        name="Blue", rows=20, seats_in_row=20
-    )
+    cinema_hall = CinemaHall.objects.create(name="Blue", rows=20, seats_in_row=20)
 
     defaults = {
         "show_time": "2022-06-02 14:00:00",
```

Comment:
Bug: The `show_time` is hard-coded. It is better to generate it dynamically to avoid future test failures if the date has passed. You can use `datetime` or `timezone` to generate a future date.